specification/operationalinsights/resource-manager/Microsoft.OperationalInsights/preview/2015-11-01-preview/OperationalInsights.json

```diff
@@ -1697,6 +1697,14 @@
           "minimum": -1,
           "maximum": 730,
           "description": "The workspace data retention in days. -1 means Unlimited retention for the Unlimited Sku. 730 days is the maximum allowed for all other Skus. "
+        },
+        "publicNetworkAccessForIngestion": {
```

Comment:
>"publicNetworkAccessForIngestion": { [](start = 8, length = 36)

these need to go into a new api-version.  Existing clients for this api-version will not be sending this which would revert to "Enabled" due to the default behavior if somebody had set it to "Disabled"

Reply:
Hi Chris,
This is a nullable enum, in other words, if property is not set we take the existing one. So in the case you mentioned, if user sets Disabled, and then user doesn't set, it will still be Disabled.
If there's other way we should set this in the swagger, please let me know.
Thanks, Arik.

Reply:
Thats not how PUT semantics should work.  If the property is not present it should revert to default.  Other RPs have added this in a new api-version, that pattern should be followed here as well.

Reply:
ok Chris, I added a new api version. Can you please review ?